### ISSUE TITLE
I've created an automatic .cpp generator

### ISSUE THREAD

**Issue by adamff-dev** [Thecakeisgit/LocaleKeyboard.js#34]
The software I've made generates a Keyboard.cpp according to our locale chosen.
This is the main idea: https://www.youtube.com/watch?v=gxE7GKLcC2w
- You can use this software in Linux: https://github.com/BlueArduino20/LocaleKeyboard.SH
- Or Windows: https://github.com/BlueArduino20/LocaleKeyboard.BAT

**Comment by lowlevl** [comment]
Hi, @BlueArduino20, thanks for doing this, but why ? The LocaleKeyboard.js does the same thing, why not creating an index.html ? I don't really understand why you did this ;D
Thanks anyway...

**Comment by adamff-dev** [comment]
I think that it's easier using the command line (And more friendly for pentesters)

**Comment by lowlevl** [comment]
Pentesters could use npm :D

**Comment by adamff-dev** [comment]
Right, but if we've got more ways to do something, better. It's about variety :). Anyway, thank you for doing this repository!

**Comment by lowlevl** [comment]
No problem ^^
Maybe improving your project could be great, because I saw you hard-coded some things like locale selection :S

**Comment by adamff-dev** [comment]
Take a look to the new scripts, I've modified them.

**Comment by exanor** [comment]
n
